omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -52,6 +52,9 @@
             # load isn't supported by windows
             delete "#{conf_dir}/load.d"
 
+            # service_discovery isn't supported by windows
+            delete "#{conf_dir}/service_discovery.d"
```

Comment:
❓ small question to have more context

windows paths are to the form here\a\windows\path
or here it seems to always be in the shape of a linux path ? is it intended ?

Reply:
I'm not sure, I followed the other examples in the file

Reply:
let me ask to my team, then I get back to you to give a more precise answer for that if it works for you ?

Reply:
Thank you!

Reply:
Got feedbacks from my team and it seems to be ok 👍🏻

Reply:
TY!